.github/workflows/tests.yml

```diff
@@ -17,14 +17,29 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+      - uses: actions/cache@v2
+        with:
+          path: Pods
+          key: ${{ runner.os }}-pods-${{ hashFiles('**/Podfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-pods-
       - name: Bundler
-        run: bundle install
+        run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
       - name: Pod install
         run: bundle exec pod install
       - name: Run tests
         run: bundle exec fastlane tests
       - name: Codecov
         uses: codecov/codecov-action@v1.0.13
+      - run: find . -type d
```

Comment:
A to w jakim celu?

Reply:
Szukam katalogów z cachem, bo widzę, że nie działa cache dla Pod.

Reply:
Już skasowaałem.